source/hassio/zwave.markdown

```diff
@@ -42,6 +42,14 @@ zha:
   database_path: /config/zigbee.db
 ```
 
+### {% linkable_title INTEL NUC %}
+
+If you have INTEL NUC and host Ubuntu, on some system's the Z-Wave Aeotec stick is not starting up correctly - no Z-Wave nodes on Home Assistant start. Then you have to disable ModemManager:
```

Comment:
I'd suggest expanding this a bit, to explain a bit more about why, something along the lines of:

If you are using the alternative install of Hass.io on Ubuntu and you have ModemManager installed, then this will interfere with the use of your Z-Wave stick. You have to disable ModemManager:

Reply:
Addressed